app/components/TracksCard/index.js

```diff
@@ -4,37 +4,209 @@
  *
  */
 
-import React from 'react';
+import React, { useRef, useState } from 'react';
 import PropTypes from 'prop-types';
 import styled from 'styled-components';
 import { Card, Image, Button } from 'antd';
 import { isEmpty } from 'lodash';
 import If from '@components/If';
+import { colors } from '@themes';
+// import { useHistory } from 'react-router-dom';
```

Comment:
remove if not required

Reply:
this is needed to redirect to track details page